pkg/pipeline/ingest/ingest_kafka.go

```diff
@@ -91,7 +91,21 @@ func (ingestK *ingestKafka) kafkaListener() {
 
 }
 
-func processRecordDelay(record config.GenericMap) {
+func (ingestK *ingestKafka) processPacketMetrics(record config.GenericMap) {
```

Comment:
I had to create a dedicate function for kafka because the field is a float64 while it is an uint64 in the other ingest.

We need to investigate this, may be for all fields.